fidesctl/setup.py

```diff
@@ -20,15 +20,15 @@
 redshift_connector = "sqlalchemy-redshift==0.8.8"
 fastapi = "fastapi==0.68"
 uvicorn = "uvicorn==0.15"
-aws_connector = "boto3==1.20.54"
+aws_connector = "boto3==1.21.33"
```

Comment:
IMO the changes in this file really belong in their own PR. Bumping a dependency version should be documented/justified.